examples/pipeline/2_pre_and_post_processors.py

```diff
@@ -54,7 +52,7 @@ def pong_processor(ctx: Context):
 
 
 if __name__ == "__main__":
-    if get_auto_arg():
+    if should_auto_execute():
```

Comment:
we can rename func and add comments
```
if is_pytest(): # condition check if the current runtime is used for verification tests
```

Reply:
Now none of our test use `-a` console argument. Should we keep it for future usage or maybe we shall remove it right away?